cmd/age/age.go

```diff
@@ -169,11 +177,47 @@ func passphrasePrompt() (string, error) {
 func encryptKeys(keys []string, in io.Reader, out io.Writer, armor bool) {
 	var recipients []age.Recipient
 	for _, arg := range keys {
-		r, err := parseRecipient(arg)
+		url, err := url.Parse(arg)
```

Comment:
Rename the variable to something other than `url` so it won't clash with the package name.